@Types/ITechRecord.d.ts

```diff
@@ -77,3 +81,55 @@ interface Axle {
         tyreCode: number
     };
 }
+
+export interface AdrDetails {
+    vehicleDetails: VehicleDetails;
+    permittedDangerousGoods: PERMITTED_DANGEROUS_GOODS[];
+    additionalExaminerNotes: string;
+    applicantDetails: ApplicantDetails;
+    memosApply: MEMOS_APPLY[];
+    additionalNotes: AdditionalNotes;
+    tank: Tank;
+}
+
+export interface AdditionalNotes {
+    number: number;
+    guidanceNotes: string[];
+}
+
+export interface ApplicantDetails {
+    name: string;
+    street: string;
+    town: string;
+    city: string;
+    postcode: string;
+}
+
+export interface Tank {
+    tankDetails: TankDetails;
+    tankStatement: TankStatement;
+}
+
+export interface TankDetails {
+    tankManufacturer: string;
+    tc2IntermediateApprovalNo: string;
+    tc2IntermediateExpiryDate: Date;
+    tc3PeriodicNumber: string;
+    tc3PeriodicExpiryDate: Date;
+    yearOfManufacture: string;
+    tankCode: string;
+    specialProvisions: string;
+    tankManufacturerSerialNo: string;
+    tankTypeAppNo: string;
+}
+
+export interface TankStatement {
+    substancesPermitted: SUBSTANCES_PERMITTED;
+    statement: string | null; // statement and productList are mutually exclusive
+    productList: string | null;
```

Comment:
As discussed with Bogdan, a runtime check for this business rule was not necessary